js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode.tsx

```diff
@@ -63,7 +68,18 @@ export const AssetNode = React.memo(({definition, selected}: Props) => {
             <AssetNodeChecksRow definition={definition} liveData={liveData} />
           )}
         </AssetNodeBox>
-        <AssetComputeKindTag definition={definition} style={{right: -2, paddingTop: 7}} />
+        <Box style={{right: -2}} flex={{direction: 'row-reverse', gap: 8}}>
```

Comment:
the negative thing is weird? Does it even do anything? I would assume it doesn't since this element isnt positioned absolutely

Reply:
yeah good call idt it does anything